src/main/java/org/openlmis/buq/dto/remark/RemarkDto.java

```diff
@@ -32,12 +32,15 @@
 @AllArgsConstructor
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
-public class RemarkDto extends BaseDto implements Remark.Importer, Remark.Exporter {
+public final class RemarkDto extends BaseDto implements Remark.Importer, Remark.Exporter {
```

Comment:
Why `final`?

Reply:
@pwargulak I had to change it for one test to be passed as long as I remember. I was just copying integration tests from sources of funds, and the SourceOfFundDto is marked as final.

Reply:
https://jqno.nl/equalsverifier/manual/final/